static/app/stores/groupStore.tsx

```diff
@@ -60,7 +61,7 @@ interface GroupStoreDefinition extends CommonStoreDefinition<Item[]>, InternalDe
   loadInitialData: (items: Item[]) => void;
 
   onAssignTo: (changeId: string, itemId: string, data: any) => void;
-  onAssignToError: (changeId: string, itemId: string, error: Error) => void;
+  onAssignToError: (changeId: string, itemId: string, error: RequestError) => void;
```

Comment:
This was changed from `Error` to `RequestError` so that the function can access the response JSON, and thus give a different toast message based on the backend error.

Reply:
Does this have a corresponding backend change? Or do we just need to use the right type on the frontend?

Reply:
We just need to use the right type on the frontend.